palindrome.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
```

Comment:
Again, when you submit code that doesn't work, it should be _very_ thoroughly commented.